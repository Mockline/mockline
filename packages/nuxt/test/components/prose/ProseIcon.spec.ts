import { describe, it, expect } from 'vitest'
import type {TypeOf} from "zod";
// @ts-ignore
import { MProseIcon } from '#components'
import ComponentRender from "../../component-render";

describe('ProseIcon', () => {
  it.each([
    ['basic case', { props: { name: 'name' } }]
  ])('renders %s correctly', async (nameOrHtml: string, options: TypeOf<typeof MProseIcon.props>) => {
    const html = await ComponentRender(nameOrHtml, options, MProseIcon)
    expect(html).toMatchSnapshot()
  })
})
