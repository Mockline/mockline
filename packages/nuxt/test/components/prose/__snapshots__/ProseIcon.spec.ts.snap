// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ProseIcon > renders basic case correctly 1`] = `"<span class="iconify i-name size-4 shrink-0 align-sub" aria-hidden="true"></span>"`;
